lib/AstToMiniRust.ml

```diff
@@ -1562,7 +1567,7 @@ let compute_struct_info files =
     method! visit_DFunction _ _ _ _ _ ret_t _ _ _ =
       match ret_t with
       | TQualified lid -> Idents.LidSet.singleton lid
-      | _ -> Idents.LidSet.empty
+      | _ -> boxed_types
```

Comment:
seems like it'd be easier to just do `Idents.LidMap.union returned boxed_types`

otherwise looks good thanks!

Reply:
Fixed in [2ab4194](https://github.com/FStarLang/karamel/pull/549/commits/2ab419433ea0d871b6de525c7a3d34d36d4aa872). Thanks!